### ISSUE TITLE
[Accessibility]A11y_VSCode_SelectEncoding_Screenreader: NVDA does not read the placeholder present on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51828]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
NVDA 2018.1.1,JAWS18.0.5038,MAS1.3.1

**Repro Steps:**
1)Launch VSCode.
2) Navigate to Status Bar and select " Select Encoding" button.
3)Check with NVDA reading.

**Actual:**
NVDA does not read the placeholder present on the screen.

**Expected:**
NVDA should read the placeholder which is present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)

**User Impact:**
User will not able to know that there is placeholder present  on the screen.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
](url)

**Attachment for Reference:**
[[Accessibility]A11y_VSCode_SelectCoding_Screenreade.pptx](https://github.com/Microsoft/vscode/files/2100960/Accessibility.A11y_VSCode_SelectCoding_Screenreade.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51824
